src/cls/rgw/cls_rgw_client.cc

```diff
@@ -467,11 +496,6 @@ int cls_rgw_clear_olh(IoCtx& io_ctx, librados::ObjectWriteOperation& op, string&
   encode(call, in);
   int op_ret;
   op.exec(RGW_CLASS, RGW_BUCKET_CLEAR_OLH, in, &out, &op_ret);
-  int r = io_ctx.operate(oid, &op);
```

Comment:
this is where the op_ret part came from, but you can just remove the 'out' and 'op_ret' arguments to op.exec(), since CLEAR_OLH is a write op and doesn't return anything:
```c++
op.exec(RGW_CLASS, RGW_BUCKET_CLEAR_OLH, in);
```